Sources/BottomSheet/BottomSheet.swift

```diff
@@ -91,7 +91,10 @@ public struct BottomSheet<Content: View>: View {
         .gesture(
             DragGesture()
                 .onChanged({ (value) in
-                    
+                    // ignore the swipe-up gesture which could collide with the iOS-global "go to homescreen"-gesture
+                    let isSwipeUp = value.startLocation.y > value.location.y
+                    if isPresented == false, isSwipeUp { return }
```

Comment:
!isPresent is better?

Reply:
And since it's an early return, try with guard?

Reply:
Good point, we can just ignore drag events while the sheet is not presented. PR is updated.